kirin.proto

```diff
@@ -33,4 +33,5 @@ extend transit_realtime.TripUpdate.StopTimeEvent {
 
 extend transit_realtime.VehicleDescriptor {
     optional string physical_mode_id = 1000;
+    optional string label = 1001;
```

Comment:
There's already a VehicleDescriptor.label in gtfsrt: https://github.com/CanalTP/chaos-proto/blob/master/gtfs-realtime.proto#L514

Also, to me, this headsign is not related to Vehicle but to Trip.
So I'd add a TripDescriptor.headsign and use it in that case.